control-plane/catalog/to-k8s/sink.go

```diff
@@ -38,7 +38,7 @@ type Sink interface {
 
 // K8SSink is a Sink implementation that registers services with Kubernetes.
 //
-// K8SSink also implements controller.Resource and is meant to run as a K8S
+// K8SSink also implements controller. Resource and is meant to run as a K8S
```

Comment:
I think this is saying `controller.Resource` and is not a sentence

Reply:
Oh. Oops.